update.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+flatpak run \
+  --filesystem=$(pwd) \
+  org.flathub.flatpak-external-data-checker \
+  com.adilhanney.ricochlime.json \
+  --update
```

Comment:
this will be done automatically on github....

Reply:
I just thought it'd be handy to have the command written down somewhere. If you want me to remove it I can do

Reply:
it doesn't matter.